cherrypy/test/test_http.py

```diff
@@ -160,10 +180,13 @@ def test_post_filename_with_special_characters(self):
         '''Testing that we can handle filenames with special characters. This
         was reported as a bug in:
            https://github.com/cherrypy/cherrypy/issues/1146/
-           https://github.com/cherrypy/cherrypy/issues/1397'''
+           https://github.com/cherrypy/cherrypy/issues/1397
+           https://github.com/cherrypy/cherrypy/issues/1694'''
         # We'll upload a bunch of files with differing names.
         fnames = ['boop.csv', 'foo, bar.csv', 'bar, xxxx.csv', 'file"name.csv',
-                  'file;name.csv', 'file; name.csv']
+                  'file;name.csv', 'file; name.csv',
+                  ('test_łóąä.txt', 'utf-8')
```

Comment:
m/b `u'test_łóąä.txt'`?

Reply:
So I wasn't sure but I think it may be better to pass str not unicode object here
and this is why I put "# coding: utf-8" at the top of the file.